src/app/branded/analytics/branded-analytics.factory.js

```diff
@@ -37,6 +37,7 @@ function generateEcommerce (siebelTransactionId, jobId) {
       price: amountPaid,
       quantity: '1',
       recurring_date: item.giftStartDate ? item.giftStartDate.format('MMMM D, YYYY') : undefined,
+      campaign_code: item.config.CAMPAIGN_CODE || undefined,
```

Comment:
Is `item.config` defined for branded checkout? I'm not sure.

Reply:
Yes https://github.com/CruGlobal/give-web/blob/d30faafd08b4a1f40463a5bb4fcc1a65eb6167f6/src/app/branded/step-1/branded-checkout-step-1.component.js#L36